src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const partnerMessage = person.isMarried ? (
+    `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+  ) : (
+    <p className="Person__partner">I am not married</p>
```

Comment:
Why you use `<p className="Person__partner">` here? It should be just string message